src/test/regress/expected/multi_repair_shards.out

```diff
@@ -1,4 +1,6 @@
 ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 820000;
+SELECT groupid AS worker_2_group FROM pg_dist_node WHERE nodeport=:worker_2_port \gset
+SELECT groupid AS worker_1_group FROM pg_dist_node WHERE nodeport=:worker_1_port \gset
```

Comment:
I would be kind of nice if we just set :worker_1_group to 1, and :worker_2_group to 2 for all tests and would then be able to use `SELECT master_add_node('localhost', :worker_1_port, group_id := :worker_1_group)` such that the group IDs are always known.